tests/integration/conftest.py

```diff
@@ -98,8 +98,13 @@ def sql_fetch_all(sql_backend):
 
 
 @pytest.fixture
-def make_ucx_group(make_random, make_group, make_acc_group, make_user):
-    def inner(workspace_group_name=None, account_group_name=None, **kwargs):
+def make_ucx_group(acc, ws, make_random, make_group, make_acc_group, make_user):
+    def inner(
+        workspace_group_name: str | None = None,
+        account_group_name: str | None = None,
+        wait_for_provisioning: bool = True,
```

Comment:
```suggestion
```
don't make it configurable and do it by default

Reply:
Done.